datajunction-server/datajunction_server/models/measure.py

```diff
@@ -0,0 +1,71 @@
+"""
+Models for measures.
+"""
+import enum
+from typing import TYPE_CHECKING, List, Optional
+
+from sqlalchemy.sql.schema import Column as SqlaColumn
+from sqlalchemy.types import Enum
+from sqlmodel import Field, Relationship
+
+from datajunction_server.models.base import BaseSQLModel
+
+if TYPE_CHECKING:
+    from datajunction_server.models import Column
+
+
+class AggregationRule(str, enum.Enum):
+    """
+    Type of allowed aggregation for a given measure.
+    """
+
+    ADDITIVE = "additive"
+    NON_ADDITIVE = "non-additive"
+    SEMI_ADDITIVE = "semi-additive"
+
+
+class MeasureColumn(BaseSQLModel, table=True):  # type: ignore
```

Comment:
Sorry I know I added this, but in thinking through it more, it might make more sense just to remove the link table `MeasureColumn` and keep `columns: List["Column"]` on `Measure` and `measure: Measure` on `Column`. Since it will be a one(-measure)-to-many(-columns) relationship, we can just store the measure_id foreign key on `Column`, and have a `measure` field on `Column` that's backpopulated.

Reply:
Hmm...  I see. Yes that should work and it should be just as fast for both searching both ways.

Reply:
@shangyian I made some changes but I am not sure this is exactly what we need. Let's go over this tomorrow.

Reply:
@agorajek nice, these changes look good to me! I just pulled it down and played around with it by adding a test -- seems to work as expected. But we can discuss your concerns tomorrow nonetheless

Reply:
Thanks for the unit test @shangyian !